sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2268,6 +2269,7 @@ def hpat_pandas_series_corr_impl(self, other, method='pearson', min_periods=None
 
 
 @sdc_overload_method(SeriesType, 'head')
+@vtune_profiling_overload(name_handle="Series_head")
```

Comment:
It should be part of `sdc_overload_method` decorator. We can extract type name and method name from where.

Reply:
You actually could call this decorator inside `sdc_overload_method`